config/config/common-config.yaml

```diff
@@ -158,6 +158,12 @@ prometheus:
   tolerations: []
   affinity: {}
   nodeSelector: {}
+  ## Predictive alert if the resource usage will hit the set percentage in 3 days
+  predictLinear:
+    enabled: true
+    disklimit: 66
+    usagelimit: 66
+    requestslimit: 85
```

Comment:
Did all of these work (not alert) on our regular cluster sizing?

Reply:
I will double check and update them after I remove the alerts for master

Reply:
done + added a screenshot